skietbaan_fe/src/components/Login.js

```diff
@@ -1,98 +1,135 @@
 import React, { Component } from 'react';
 import {
-  Container, Col, Form,
-  FormGroup, Label, Input,
-  Button,
+  Container, Col,
+  FormGroup, Label, Input,Button,Form
 } from 'reactstrap';
 import '../components/LoginStyles.css';
 
+  function validatePassword(str) {
+      var re = /(?=.*\d)(?=.*[a-z]).{6,}/;
```

Comment:
Variable declared with `var` have a global scope. This can easily lead to weird bugs in your app where variables with the same name override each other. 
Rather use `let` (i.e. `let name = 'Steve';`), which has block scope. or `const` for variables that won't change value and to make the value is never overridden by mistake.

Reply:
Ok noted, thanks